[RM Chap8] Multi-Cloud Interactions Model Summary Table (#2824)
cntt-n/CNTT: doc/ref_model/chapters/chapter08.rst

Requirements, Reference Architecture & Industry Standards Intersect
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The Communcations Service Provider (CSP) is both a provider and consumer of Cloud based services. When the CSP is acting as:

> - consumer of cloud services, the typical consideration is total cost of ownership as the consumption is usually to support internal business operations: BSS/OSS systems;

Reviewer: Doesn't show as bullets any more.

Author: @ulikleber experience/testing has shown that you need to have a blank line before the start of the bulleted list. I will update. Thank you for feedback.

The interactions for this are outlined in the "Multi-Cloud Interactions Model", however, to realise this, the CSP will need to adopt and sponsor a set of standards that are necessary to support these interactions. The identification of existing appliable standards and gaps across the interactions needs to be completed. As a first step, the following criteria for inclusion of a standard/technology is defined. These standards/technologies must:

- provide capabilities that are necessary to achieve hybrid multi-cloud vision and the multi-cloud interactions
- be already mature Open Standards that have either been adopted or nurtured by recognised bodies with the telecommunications industry (e.g. ITU, ETSI, TMForum, GSMA, 3GPP, ISO and national Standards Organiations , (ANSI etc,) NIST)
- have reference implementations or an active open source project/s or consortia providing implementations (e.g., CNCF. Open Infra)
- allow the CSP to source delivery and support services based on these from multiple vendors
- allow the CSP to actively contribute to and request capabilities / coverage of the standard / technology
- not  be the sole proprietary property of a vendor / company

> - not be focused on "Transactions / Conversations" or "User / Data Plane" standards (typically IETF, IEEE, MEF / Carrier Ethernet etc.)

Reviewer: Not sure how to deal with requirements against standards. RM should set requirements against platforms. So a requirement in RM could be to follow certain standards which provide capabilites...., ..., and give a list of the standards to follow. But requiring something from an SDO, I don't know.

Author: @ulikleber initial ambition was to populate table of applicable "standards", that would be recommended towards helping CSP to meet objectives of standing up and operating a multi-cloud environment and identify gaps that need to be addressed.  For this release (moselle) the ambition has been curtailed to just getting the criteria for inclusion of such "standards" (which was identified as required toward the end of prior release (lakelse)). If we can get to agreement on what is already covered (beyond OpenStack & Kubernetes) and what is missing, then we can agree what should be approach to handling the "gap" items. I agree that Anuket cannot mandate that some organisation should take on responsibility of covering the gap. I think in identify the gap that Anuket (RM) could help elaborating what this is and provide statement of requirement/need which can be use as input to CSP to "consider" (and maybe even tackle ;-) ). So discussion point rather than resolution.

Requirements, Reference Architecture & Industry Standards Intersect
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The Communcations Service Provider (CSP) is both a provider and consumer of Cloud based services. When the CSP is acting as:

Author: @CsatariGergely could you please point me to formatting guide, as I am not sure if this is a manual thing or something that you setup in github. Thank you.

> The Communcations Service Provider (CSP) is both a provider and consumer of Cloud based services. When the CSP is acting as:

Reviewer 2: We have only an agreement in the TSC to wrap at 120 chars. We do not have a formatting guide. The plan is to turn on rst lining in the pr gating process and we are working on the wrapping of all the documents. 
On the other hand I agree that a written formatting guide would be very good on top of the codified one. @scottsteinbrueck maybe we can discuss this in one of our Doc Evangelist meetings.

> The Communcations Service Provider (CSP) is both a provider and consumer of Cloud based services. When the CSP is acting as:

Author: @CsatariGergely wrapping does as part of editorial/review conflict resolution.

-  consumer, in which case the typical consideration is total cost of ownership as the consumption is to usually to support internal business operations: BSS/OSS systems;
-  provider of cloud services, through operation of their own cloud or reselling of cloud services, in which case the typical consideration is margin (cost to offer services vs income received).
These two stances will drive differing approaches to how a CSP would look to manage how it interacts within a Multi-Cloud environment.

> As a consumer of cloud to support internal Business operations and BSS/OSS focus is on meeting the applications needs of organisation which historically came with need to operate support the infrastructure needs of these. This resulted in split of CIO organisation into Delivery and Operations groups. At the same time that CIO application workloads have been moving to SaaS Cloud Providers, CTO Network Systems have been moving from running on dedicated infrastructure to being virtualised and IMS, 3GPP (4G & 5G) functions, IP Routers and Firewalls are being provided as VNFs and CNFs.

Author: @pgoyal01 thank you, I have have updated wording and also added wording on VNF/CNF also being candidate for deployment on public and private clouds.

> As outlined in section "8.2.2 Telco Cloud" the result is that the future CSP "network" is a set of distributed NFVi's (Cloud Infrastructure) which will be connected to Cloud Providers and hence the "Hybrid Multi-Cloud" and the need for CSP to be able to support this is both inevitable and essential.

Reviewer 3: ```suggestion
As outlined in section "8.2.2 Telco Cloud", the result is that the CSP "network" is now an interconnected set of distributed NFVi's (Cloud Infrastructure) supported by different Cloud Providers, including the CSP, and, hence, the term "Hybrid Multi-Cloud", and the need for the CSP to be able to support and utilize this interconnected cloud is both inevitable and essential.
```

> As outlined in section "8.2.2 Telco Cloud", the result is that the CSP "network" is now an interconnected set of distributed NFVi's (Cloud Infrastructure) supported by different Cloud Providers, including the CSP, and, hence, the term "Hybrid Multi-Cloud", and the need for the CSP to be able to support and utilize this interconnected cloud is both inevitable and essential.

Reviewer 2: Please use "cloud infrstructure" instead of NFVI.

Author: @CsatariGergely done, thank you for feedback.

> So as a consumer and provider of Cloud Services the CSP will continue to need to build and manage it own Cloud Infrastructure as well as provide network orchestration solution to management interconnectivity across its own and other Cloud Providers. The interactions for this are outlined in the "Multi-Cloud Interactions Model", however to realise this the CSP will need to adopt and sponsor a set of standards that are nessasary to support the interactions.

Reviewer 3: ```suggestion
So as a consumer and provider of Cloud Services, the CSP will continue to need to build and manage its own Cloud Infrastructure as well as provide 
- cloud orchestration solutions to orchestrate the utilisation of cloud services and capabilities from its own and other Cloud Providers;
- network orchestration solutions to manage the interconnectivity across its own and other Cloud Provider networks. 

The interactions for this are outlined in the "Multi-Cloud Interactions Model", however, to realise this, the CSP will need to adopt and sponsor a set of standards that are necessary to support these interactions.
```

Author: @pgoyal01 thank you. Corrections and updates have been applied.

> The follow summary table aims to identify what standards and/or technologies that are seen are applicable.

Reviewer 3: ```suggestion
The following summary table aims to identify the applicable standards and/or technologies.
```

- network orchestration solutions to manage the interconnectivity across its own and other Cloud Provider networks. 

The interactions for this are outlined in the "Multi-Cloud Interactions Model", however, to realise this, the CSP will need to adopt and sponsor a set of standards that are necessary to support these interactions. The identification of existing appliable standards and gaps across the interactions needs to be completed. As a first step, the following criteria for inclusion of a standard/technology is defined. These standards/technologies must:

> -  provide capabilities that are necessary to achieve hybrid multi-cloud vision and the multi-cloud interactions

Reviewer 2: Please use only one space after the "-"

Author: @CsatariGergely I have done pass through and removed all bulleted lists with double white space. Thank you for feedback.

> -  Are already mature Open Standards that have either been adopted or nurtured by recognised bodies with the telecommunications industry (e.g. ITU, ETSI, TMForum, GSMA, 3GPP, ISO and national Standards Organiations , (ANSI etc,) NIST)

Reviewer 3: ```suggestion
-  Be mature Open Standards that have either been adopted or nurtured by recognised bodies with the telecommunications industry (e.g., ITU, ETSI, TMForum, GSMA, 3GPP, ISO, and national Standards Organisations , (ANSI etc.), NIST)
```

> -  There are reference implementations or an active open source project/s or consortia providing implementations (CNCF. Open Infra)

Reviewer 3: ```suggestion
-  Specify reference implementations or an active open source project/s or consortia providing implementations (e.g., CNCF. Open Infra)
```

Author: @pgoyal01 I have updated wording to make it clearer and also update the list so it consistently has lower case starting word. Thank you for feedback.

> -  It is possible to source delivery and support services from multiple vendors

Reviewer 3: ```suggestion
-  Provide source delivery and support services from multiple vendors
```

Author: @pgoyal01 I have updated word to improve readability. Thank you for feedback.

> -  It is possible for CSP to actively contribute to and request capabilities / coverage of the standard / technology

Reviewer 3: ```suggestion
-  Allow the CSP to actively contribute to and request capabilities / coverage of the standard / technology
```

> -  It is not sole proprietary property of a vendor / company

Reviewer 3: ```suggestion
-  Not  be the sole proprietary property of a vendor / company
```

> -  Is not focused on "Transactions / Conversations" User / Data Plane standards (typically IETF, IEEE, MEF / Carrier Ethernet etc)

Reviewer 3: ```suggestion
-  Not be focused on "Transactions / Conversations" or "User / Data Plane" standards (typically IETF, IEEE, MEF / Carrier Ethernet etc.)
```

-  provide capabilities that are necessary to achieve hybrid multi-cloud vision and the multi-cloud interactions
-  be already mature Open Standards that have either been adopted or nurtured by recognised bodies with the telecommunications industry (e.g. ITU, ETSI, TMForum, GSMA, 3GPP, ISO and national Standards Organiations , (ANSI etc,) NIST)

> -  have reference implementations or an active open source project/s or consortia providing implementations (e.g., CNCF. Open InfraCNCF)

Author: @CsatariGergely corrected with new commit, thank you for review.
